### ISSUE TITLE
Customize the base directory ~/.zprezto (global installation in /etc/zsh)

### ISSUE THREAD

**Issue by kaluzki** [sorin-ionescu/prezto#1025]
Hey,

I would like to use zsh with prezto as default shell for several users (1000 and root at least) in my virtual machines. 
But I want to clone the repo only once. Currently it is not possible because the passage "${ZDOTDIR:-$HOME}/.zprezto/" 
is used in several places (most in the pmodload function).

I fixed it with 576f2b86af07af8a4fbfbc1484bf7440bf2c0e82 in my fork. Is there a chance to push it into the upstream?

**Comment by Eriner** [comment]
An alternitive solution would be to use a sed, and install it globally (as a package) as the [arch AUR package does](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=prezto-git).

Your use case is a rather specific one.

**Comment by kaluzki** [comment]
Thanks for the quick response. Actually I'm not a big fan of patching.
I would prefer just to clone the repo and keep it as close to standard as possible.

Also your "build" function seems to benefit from this change:

```
echo "ZPREZTODIR=/usr/lib/prezto" > "$srcdir/etc/zsh/zshrc"
echo "source \${ZPREZTODIR}/init.zsh" >> "$srcdir/etc/zsh/zshrc"
```
and no seds anymore ;-)

**Comment by Eriner** [comment]
That's not the point. _most_ people just use this on a per-user basis, and clone the repo. If you're going to install it globally, either sed the files, or apply your patch in a separate fork. The point is, this doesn't apply to most users.

**Comment by kaluzki** [comment]
It's a pity, this change would make my life easier. But I understand your position. Thanks anyway.

**Comment by sorin-ionescu** [comment]
Duplicate of #920, #761, #271.
